extensions/wikia/Security/classes/Utils.class.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Wikia\Security;
```

Comment:
@macbre this namespace should be in `lib/Wikia/src`. We have been trying to consolidated all of the `Wikia` namespace there so its PSR-* compliant. This also removes the need to alter the `$wgAutoloadClasses` map.